### PR TITLE
Truncate test db

### PR DESCRIPTION
**Pull request by jmcg015**
Truncates test DB before each test

### CODE REVIEW

File: models/post.js [hunk 1]
```diff
@@ -2,8 +2,17 @@ const mongoose = require("mongoose");
...
   message: String,
+  createdAt: {
+    type: Date,
+    immutable: true,
+    default: () => Date.now(),
+  },
 });
 
+PostSchema.methods.formatDate = function() {
+  return this.createdAt.toString().substring(0,10);
```
Comment (PatrickReynoldsCoding) [inline models/post.js:13]: toString() not necessary?

**Review by PatrickReynoldsCoding — APPROVED** [review]
Looks great. Nice work!

File: spec/mongodb_helper.js [hunk 2]
```diff
@@ -6,6 +6,12 @@ beforeAll(function (done) {
     useUnifiedTopology: true,
   });
 
+  const collections = mongoose.connection.collections;
```
Comment (brownc2) [inline spec/mongodb_helper.js:9]: Amazing work

**Review by brownc2 — COMMENTED** [review]
Amazing work guys

**Review by brownc2 — APPROVED** [review]

**Pull request merged**